mv /data/rene/Debian/Pakete/LibreOffice/libreoffice/libreoffice-6.0.0.3/workdir/Dep/LinkTarget/CppunitTest/libtest_sw_uiwriter.so.d.tmp /data/rene/Debian/Pakete/LibreOffice/libreoffice/libreoffice-6.0.0.3/workdir/Dep/LinkTarget/CppunitTest/libtest_sw_uiwriter.so.d
S=/data/rene/Debian/Pakete/LibreOffice/libreoffice/libreoffice-6.0.0.3 && I=$S/instdir && W=$S/workdir &&  x86_64-linux-gnu-g++ -shared -Wl,-z,noexecstack   -Wl,-z,origin '-Wl,-rpath,$ORIGIN/../Library' -Wl,-rpath-link,$I/program -Wl,-z,defs -Wl,-rpath-link,/lib:/usr/lib -Wl,-z,combreloc  -Wl,--hash-style=gnu  -Wl,--dynamic-list-cpp-new -Wl,--dynamic-list-cpp-typeinfo -Wl,-Bsymbolic-functions  -L$W/LinkTarget/StaticLibrary -L$I/sdk/lib  -L$I/program  -L$I/program  -L$W/LinkTarget/Library -Wl,-z,relro    $W/CxxObject/sw/qa/extras/uiwriter/uiwriter.o      -Wl,--start-group    -lcppunit  -lxml2 -Wl,--end-group -Wl,--no-as-needed -lmergedlo -luno_cppu -luno_cppuhelpergcc3 -lmswordlo -luno_sal -lswlo -ltest -lunotest  -o $W/LinkTarget/CppunitTest/libtest_sw_uiwriter.so 
/data/rene/Debian/Pakete/LibreOffice/libreoffice/libreoffice-6.0.0.3/workdir/CxxObject/sw/qa/extras/uiwriter/uiwriter.o: In Funktion »SwUiWriterTest::addTestsToSuite(CppUnit::TestSuiteBuilderContextBase&)«:
././sw/qa/extras/uiwriter/uiwriter.cxx:461: Warnung: undefinierter Verweis auf »SwUiWriterTest::testTdf114536()«
collect2: error: ld returned 1 exit status
/data/rene/Debian/Pakete/LibreOffice/libreoffice/libreoffice-6.0.0.3/solenv/gbuild/LinkTarget.mk:627: die Regel für Ziel „/data/rene/Debian/Pakete/LibreOffice/libreoffice/libreoffice-6.0.0.3/workdir/LinkTarget/CppunitTest/libtest_sw_uiwriter.so“ scheiterte

whyever... it is there.

--- a/sw/qa/extras/uiwriter/uiwriter.cxx-old	2018-01-27 10:23:35.652529748 +0000
+++ b/sw/qa/extras/uiwriter/uiwriter.cxx	2018-01-27 10:23:39.660660005 +0000
@@ -458,7 +458,7 @@
     CPPUNIT_TEST(testTdf99689TableOfTables);
     CPPUNIT_TEST(testTdf113790);
 //    CPPUNIT_TEST(testTdf114306);
-    CPPUNIT_TEST(testTdf114536);
+//    CPPUNIT_TEST(testTdf114536);
     CPPUNIT_TEST_SUITE_END();
 
 private:
